search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -91,18 +92,48 @@ function displayTopResultsForCurrentTrend(countryCode) {
         countryData[0].interest + '</b><br>';
     resultElement.innerHTML += '<b>Average Sentiment Score: ' + 
         countryData[0].averageSentiment.toFixed(1) + '</b><br>';
-
+    
     // Get search results for the specified country.
     let results = countryData[0].results;
     for (let i = 0; i < results.length; i++) {
-      // i + 1 shows the index for each search result. 
-      resultElement.innerHTML += (i + 1).toString() + '. ' + '<a href=' + 
-          results[i].link + '>' + results[i].htmlTitle + '</a><br>' + 
-          results[i].snippet + '<br>' + '<i>Sentiment Score: ' + 
-          results[i].score.toFixed(1) + '</i><br>';
+      await fetch('/sentiment-words/' + results[i].title + results[i].snippet)
+          .then(resultsJsonArray => resultsJsonArray.json())
+          .then(sentimentWordsResult => {
+            // i + 1 shows the index for each search result.
+            resultElement.innerHTML += (i + 1).toString() + '. ' + '<a href=' + 
+                results[i].link + '>' + results[i].htmlTitle + '</a><br>';
+            // Check to make sure that there is detection of positive and 
+            // negative words from the Node.js sentiment API.
```

Comment:
Prefer active voice over passive, and to the point :)

```suggestion
            // Check that positive and negative words are detected by the Node.js sentiment API.
```